src/plugins/variables-plugin.tsx

```diff
@@ -70,6 +76,7 @@ function getDialogValuesFromNode(node?: Inline) {
 const kSpanTag = "span";
 
 export function VariablesPlugin(variables: Record<string, number>): HtmlSerializablePlugin {
+  const [presetVariables] = useState(variables);
```

Comment:
I agree with @scytacki that you shouldn't use `useState()` here since this is a Slate plugin not a React functional component. I think you can probably just use:
```suggestion
  const presetVariables = clone(variables);
```